log/development.log

```diff
@@ -0,0 +1,35 @@
+  [1m[35m (100.5ms)[0m  [1m[35mCREATE TABLE "schema_migrations" ("version" character varying NOT NULL PRIMARY KEY)[0m
```

Comment:
Don't commit log files and temp files. An easy way to ignore those files is to use a .gitignore file. One example is https://github.com/github/gitignore/blob/master/Rails.gitignore, or our own at https://github.com/Wolox/rails-bootstrap/blob/master/.gitignore